struqture/tests/mappings/jordan_wigner.rs

```diff
@@ -0,0 +1,164 @@
+        .unwrap();
+    let jw_pair1 = fp1.jordan_wigner() * CalculatorComplex::new(1.0, 2.0);
+    let jw_pair2 = fp2.jordan_wigner() * CalculatorComplex::new(2.0, 1.0);
+
+    assert_eq!(fo.jordan_wigner(), jw_pair1 + jw_pair2);
+}
+
+#[test]
+fn test_jw_fermion_hamiltonian_to_spin() {
+    let mut fh = FermionHamiltonian::new();
+    let hfp1 = HermitianFermionProduct::new([1, 2], [2, 4]).unwrap();
+    let hfp2 = HermitianFermionProduct::new([1, 3], [1, 2]).unwrap();
+    fh.add_operator_product(hfp1.clone(), CalculatorComplex::new(1.0, 2.0))
+        .unwrap();
+    fh.add_operator_product(hfp2.clone(), CalculatorComplex::new(2.0, 1.0))
+        .unwrap();
+    let jw_pair1 = hfp1.jordan_wigner();
+    let jw_pair2 = hfp2.jordan_wigner();
+    let jw_pair1_hamiltonian = SpinHamiltonian::try_from(jw_pair1).unwrap();
+    let jw_pair2_hamiltonian = SpinHamiltonian::try_from(jw_pair2).unwrap();
+
+    assert_eq!(
+        fh.jordan_wigner(),
+        jw_pair1_hamiltonian * CalculatorFloat::from(1.0)
+            + jw_pair2_hamiltonian * CalculatorFloat::from(2.0)
+    );
+}
+
+#[test]
+fn test_jw_fermion_noise_operator_to_spin() {
+    let mut fno = FermionLindbladNoiseOperator::new();
+    let mut sno = SpinLindbladNoiseOperator::new();
+
+    assert_eq!(fno.jordan_wigner(), sno);
+
+    let fp = FermionProduct::new([0], [0]).unwrap();
+    fno.add_operator_product((fp.clone(), fp.clone()), CalculatorComplex::new(1.0, 0.0))
+        .unwrap();
+    let dp = DecoherenceProduct::new().z(0);
+    sno.add_operator_product((dp.clone(), dp.clone()), CalculatorComplex::new(0.25, 0.0))
+        .unwrap();
+}
+
+#[test]
+fn test_jw_fermion_systems_to_spin() {
+    // Test FermionHamiltonianSystem
+    let mut fh = FermionHamiltonian::new();
+    fh.add_operator_product(
+        HermitianFermionProduct::new([1], [2]).unwrap(),
+        CalculatorComplex::new(1.0, 2.0),
+    )
+    .unwrap();
+    let fhs = FermionHamiltonianSystem::from_hamiltonian(fh.clone(), Some(5)).unwrap();
+    let sh = fh.jordan_wigner();
+    let shs = SpinHamiltonianSystem::from_hamiltonian(sh, Some(5)).unwrap();
+
+    assert_eq!(fhs.jordan_wigner(), shs);
+
+    // Test FermionLindbladNoiseSystem
+    let mut fno = FermionLindbladNoiseOperator::new();
+    let fp1 = FermionProduct::new([1], [2]).unwrap();
+    let fp2 = FermionProduct::new([2], [3]).unwrap();
+    fno.add_operator_product((fp1, fp2), CalculatorComplex::new(1.0, 2.0))
+        .unwrap();
+    let fns = FermionLindbladNoiseSystem::from_operator(fno.clone(), Some(5)).unwrap();
+    let sno = fno.jordan_wigner();
+    let sns = SpinLindbladNoiseSystem::from_operator(sno, Some(5)).unwrap();
+
+    assert_eq!(fns.jordan_wigner(), sns);
+
+    // Test FermionLindbladOpenSystem
+    let sos = SpinLindbladOpenSystem::group(shs, sns).unwrap();
+    let fos = FermionLindbladOpenSystem::group(fhs, fns).unwrap();
+
+    assert_eq!(fos.jordan_wigner(), sos);
+}
```

Comment:
There are no tests for FermionSystem, is this intentional? I noticed it hasn't been modified, was it forgotten or is there a reason FermionOperator has been modified and not FermionSystem?